xarray/datatree_/docs/source/terminology.rst

```diff
@@ -0,0 +1,34 @@
+.. currentmodule:: datatree
```

Comment:
move the core of this page to https://docs.xarray.dev/en/stable/user-guide/terminology.html